docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -36,28 +36,23 @@ depending on the desired :ref:`retry policy <fte-retry-policy>`.
 
 .. warning::
 
-  Setting ``retry-policy`` disables :ref:`write operations
-  <sql-write-operations>` with connectors that do not support fault-tolerant
```

Comment:
So now this expands towards read and write basically? And that info moves to the connector docs.. right?

Reply:
Correct. @hashhar described to us that it's not as clear-cut as the original text suggested, and the connector docs is a better place to go into details for each connector.